tests/LanguageServerRename/Util/OffsetExtractor.php

```diff
@@ -56,7 +47,7 @@ public function registerRange(string $name, string $openMarker, string $closeMar
         return $this;
     }
 
-    public function parse(string $source): OffsetExtractor
+    public function parse(string $source): OffsetExtractorResult
```

Comment:
note that this method no longer modifies the extractor's state, it produces a result.